includes/class-donations.php

```diff
@@ -329,7 +341,7 @@ protected static function update_donation_product( $args ) {
 	 */
 	public static function remove_donations_from_cart() {
 		$donation_settings = self::get_donation_settings();
-		if ( ! $donation_settings['created'] ) {
+		if ( ! $donation_settings['created'] || is_wp_error( self::is_woocommerce_suite_active() ) ) {
```

Comment:
This line is throwing a warning if WooCommerce is active, but the Reader Revenue platform is set to NRH:

```
Cannot modify header information - headers already sent by (output started at /srv/users/user53d5f4dc/apps/user53d5f4dc/public/wp-content/plugins/newspack-plugin/includes/class-donations.php:349)
Undefined index: products in /srv/users/user53d5f4dc/apps/user53d5f4dc/public/wp-content/plugins/newspack-plugin/includes/class-donations.php on line 349
```

Reply:
I wasn't able to reproduce this myself, but it makes total sense. https://github.com/Automattic/newspack-plugin/pull/726/commits/9405be3dcc67eca654f2c0ad89830f7621e3e8b1 makes the donation data provided for NRH and WooCommerce much more similar, which should resolve all the warnings.